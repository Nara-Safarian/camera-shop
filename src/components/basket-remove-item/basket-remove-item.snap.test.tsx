import { render } from '@testing-library/react';
import BasketRemoveItem from './basket-remove-item';

test('BasketRemoveItem snapshot should be rendered correctly', () => {
  const {container} = render(
    <BasketRemoveItem onClose={function (): void {
      throw new Error('Function not implemented.');
    } } onRemove={function (): void {
      throw new Error('Function not implemented.');
    } } isActive={false}
    />
  );
  expect(container).toMatchSnapshot();
});
