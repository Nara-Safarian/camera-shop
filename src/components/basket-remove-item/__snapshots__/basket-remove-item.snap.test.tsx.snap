// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasketRemoveItem snapshot should be rendered correctly 1`] = `
<div>
  <svg
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <symbol
      id="icon-close"
      viewBox="0 0 10 10"
    >
      <path
        clip-rule="evenodd"
        d="M1.70711 0.292893C1.31658 -0.0976311 0.683418 -0.0976311 0.292893 0.292893C-0.0976311 0.683418 -0.0976311 1.31658 0.292893 1.70711L3.58579 5L0.292893 8.29289C-0.0976311 8.68342 -0.0976311 9.31658 0.292893 9.70711C0.683418 10.0976 1.31658 10.0976 1.70711 9.70711L5 6.41421L8.29289 9.70711C8.68342 10.0976 9.31658 10.0976 9.70711 9.70711C10.0976 9.31658 10.0976 8.68342 9.70711 8.29289L6.41421 5L9.70711 1.70711C10.0976 1.31658 10.0976 0.683418 9.70711 0.292893C9.31658 -0.0976311 8.68342 -0.0976311 8.29289 0.292893L5 3.58579L1.70711 0.292893Z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </symbol>
  </svg>
  <div
    class="modal is-active"
  >
    <div
      class="modal__wrapper"
    >
      <div
        class="modal__overlay"
      />
      <div
        class="modal__content"
      >
        <p
          class="title title--h4"
        >
          Удалить этот товар?
        </p>
        <div
          class="basket-item basket-item--short"
        >
          <div
            class="basket-item__img"
          >
            <picture>
              <source
                srcset="img/content/img9.webp, img/content/img9@2x.webp 2x"
                type="image/webp"
              />
              <img
                alt="Фотоаппарат «Орлёнок»"
                height="120"
                src="img/content/img9.jpg"
                srcset="img/content/img9@2x.jpg 2x"
                width="140"
              />
            </picture>
          </div>
          <div
            class="basket-item__description"
          >
            <p
              class="basket-item__title"
            >
              Орлёнок
            </p>
            <ul
              class="basket-item__list"
            >
              <li
                class="basket-item__list-item"
              >
                <span
                  class="basket-item__article"
                >
                  Артикул:
                </span>
                 
                <span
                  class="basket-item__number"
                >
                  O78DFGSD832
                </span>
              </li>
              <li
                class="basket-item__list-item"
              >
                Плёночная фотокамера
              </li>
              <li
                class="basket-item__list-item"
              >
                Любительский уровень
              </li>
            </ul>
          </div>
        </div>
        <div
          class="modal__buttons"
        >
          <button
            class="btn btn--purple modal__btn modal__btn--half-width"
            type="button"
          >
            Удалить
          </button>
          <a
            class="btn btn--transparent modal__btn modal__btn--half-width"
            href="#"
          >
            Продолжить покупки
          </a>
        </div>
        <button
          aria-label="Закрыть попап"
          class="cross-btn"
          type="button"
        >
          <svg
            aria-hidden="true"
            height="10"
            width="10"
          >
            <use
              xlink:href="#icon-close"
            />
          </svg>
        </button>
      </div>
    </div>
  </div>
</div>
`;
